flask_monitoringdashboard/views/profiler.py

```diff
@@ -19,8 +19,10 @@ def num_profiled(endpoint_id):
 @blueprint.route('/api/profiler_table/<endpoint_id>/<offset>/<per_page>')
 @secure
 def profiler_table(endpoint_id, offset, per_page):
+    sort_by = request.args.get('sortby', 'latest')
+    response_status = request.args.get('response_status', 'all').lower()
     with session_scope() as db_session:
-        return jsonify(get_profiler_table(db_session, endpoint_id, offset, per_page))
+        return jsonify(get_profiler_table(db_session, endpoint_id, offset, per_page, sort_by, response_status))
```

Comment:
In sync with the other param being called `sort_by` maybe `response_status` should be called `filter_by`? Or maybe `filter_by_status` ?

Reply:
@mircealungu @FlyingBird95 Can you please review the changes and help me out here? Thanks.

Reply:
Dear @jangster3794, 

Thanks for the reminder,  and apologies for the slowness of my response. 

I looked again at the behavior of the system with this PR and I think it's still buggy. 
What do you think? Let me higlight the steps to replicate: 

1. Before filtering based on response code I see this: 

![Screen Shot 2020-03-04 at 3 00 05 PM](https://user-images.githubusercontent.com/464519/75886839-31c6be80-5e29-11ea-930e-d7cf9f4f2d78.png)

2. After filtering only the Failed I see this: 
- note that although there's only one request the info at the top mentions stil 24K and I still have the choice of selecting other pages

![Screen Shot 2020-03-04 at 3 00 27 PM](https://user-images.githubusercontent.com/464519/75886893-41460780-5e29-11ea-9331-0cb35346704a.png)

3. And obviously, when I select page 2 I get no request listed, but twice the navigation pannels...

![image](https://user-images.githubusercontent.com/464519/75887016-6fc3e280-5e29-11ea-8ec8-4d6be2509649.png)

Reply:
Hi,
Sorry for the late reply.

Thanks for highlighting these bugs.
I am going to start work on these bugs and update this PR in a couple of days.